.circleci/config.yml

```diff
@@ -8,28 +8,24 @@ version: 2
 jobs:
   build:
     working_directory: ~/go-path/src/github.com/keybase/kbfs
-    parallelism: 1
 
     docker:
     - image: keybaseprivate/circleci-kbfs@sha256:6e95999341a551ca9d175f497abc6f99e42583d977e26489471a86895327cd28
-      command: /sbin/init
 
     steps:
-    # Clone client.
-    - run: git clone https://github.com/keybase/client.git $GOPATH/src/github.com/keybase/client
-
-    ##
-    ## Build.
-    ##
-
     - checkout
 
     # TODO: Investigate whether we can do some caching to speed up
     # this step, e.g. of /tmp/go-android/pkg.
     - run:
-        working_directory: ../client/shared
         environment:
+          LOCAL_CLIENT: 0
```

Comment:
Would be nice if client master pushes baked the client repo into the image; then we really wouldn't have to clone at all.

Reply:
Yeah, that would be nice. I wonder if that's possible with CircleCI's workflows stuff...